### PR TITLE
[TASK] Move PHP code sniffing to GitHub actions

### PR DESCRIPTION
**Pull request by andreaswolf**
Resolves: #13

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,3 +23,25 @@ jobs:
...
+          - 7.2
+          - 7.3
+          - 7.4
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Install Composer dependencies
+      run: composer install --prefer-dist --no-progress
```
Comment (oliverklee) [inline .github/workflows/ci.yml]: The prefer-dist already is configured in the `composer.json`:
https://github.com/typo3-trainer-network/tea/blob/master/composer.json#L59

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Composer script for PHP linting
 
 ### Changed
+- Moved PHP code sniffing to GitHub actions (#13)
```
Comment (oliverklee) [inline CHANGELOG.md]: In general: Moved -> Move

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -23,3 +23,25 @@ jobs:
...
+    name: PHP Code Sniffer
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
```
Comment (oliverklee) [inline .github/workflows/ci.yml]: For sniffing, one PHP version should suffice, as the result will not depend on the PHP version running the sniffer.

File: CHANGELOG.md (hunk 2 above)
Comment (andreaswolf) [reply CHANGELOG.md]: I tend to disagree, since the headline is also past tense. But then we should also fix all existing entries.

Comment (oliverklee) [reply CHANGELOG.md]: I usually copy the Git commit message headlines into the changelog, resulting in this form.
https://chris.beams.io/posts/git-commit/

(But I'm not very passionate about the tenses in the changelog.)

Comment (andreaswolf) [reply CHANGELOG.md]: Sounds logical, let's stick to present tense then.

**Pull request merged**